lib/v2/lightnovel/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'lightNovel.us': {
+        _name: '轻之国度',
+        '.': [
+            {
+                title: '文章更新阅读',
+                docs: 'https://docs.rsshub.app/routes/annime#wen-zhang-geng-xing-yue-du',
+                source: '/',
+                target: '/lightNovel/:keywords/:security_key',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/86c458f9d80d92495e46c04ff7a3650f53390264/website/docs/routes/anime.mdx#L841
```suggestion
                target: '/lightnovel/:keywords/:security_key',
```